src/test_application/activescenes/scenarios_universetest.cpp

```diff
@@ -0,0 +1,701 @@
+/**
+ * Open Space Program
+ * Copyright © 2019-2022 Open Space Program Project
+ *
+ * MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+#include "scenarios.h"
+#include "scene_physics.h"
+#include "common_scene.h"
+#include "common_renderer_gl.h"
+#include "CameraController.h"
+
+#include "../ActiveApplication.h"
+
+#include <osp/Active/basic.h>
+#include <osp/Active/drawing.h>
+#include <osp/Active/physics.h>
+#include <osp/Active/SysHierarchy.h>
+
+#include <osp/universe/universe.h>
+#include <osp/universe/coordinates.h>
+
+#include <osp/Resource/resources.h>
+#include <osp/CommonMath.h>
+#include <osp/logging.h>
+
+#include <longeron/id_management/registry.hpp>
+
+#include <Magnum/GL/DefaultFramebuffer.h>
+
+#include <Corrade/Containers/ArrayViewStl.h>
+
+#include <newtondynamics_physics/ospnewton.h>
+#include <newtondynamics_physics/SysNewton.h>
+
+#include <random>
+
+using osp::active::ActiveEnt;
+using osp::active::active_sparse_set_t;
+using osp::active::MeshIdOwner_t;
+
+using osp::phys::EShape;
+
+using osp::Vector3;
+using osp::Vector3d;
+using osp::Quaternion;
+using osp::Quaterniond;
+using osp::Matrix3;
+using osp::Matrix4;
+
+// for the 0xrrggbb_rgbf and angle literals
+using namespace Magnum::Math::Literals;
+
+using Corrade::Containers::Array;
+
+namespace testapp::scenes
+{
+
+constexpr float gc_physTimestep = 1.0 / 60.0f;
```

Comment:
put these in an anymous namespace.

Helps the linker discard symbols that don't need to be visible outside the translation unit.